### PR TITLE
api: Returns conflict when inserting an existing namespace

### PR DESCRIPTION
**Pull request by leonardojoao**
This commit also removes the data return from the namespace, as it's
not necessary.

### CODE REVIEW

File: api/nsadm/service.go [hunk 1]
```diff
@@ -77,7 +78,14 @@ func (s *service) CreateNamespace(ctx context.Context, namespace *models.Namespa
 		namespace.MaxDevices = -1
 	}
 
-	return s.store.NamespaceCreate(ctx, namespace)
+	err := s.store.NamespaceCreate(ctx, namespace)
+	if err != nil {
+		if strings.Contains(err.Error(), "duplicate key error") {
```
Comment (leonardojoao) [inline api/nsadm/service.go:83]: I am looking for a solution for this type of verification.

**Review by otavio — DISMISSED** [review]

File: cli/main.go [hunk 2]
```diff
@@ -54,18 +54,14 @@ func main() {
...
-				ns, err := svc.NamespaceCreate(Parameters{
+				err := svc.NamespaceCreate(Parameters{
 					Namespace: args[0],
 					Username:  args[1],
 				})
 				if err != nil {
 					return err
 				}
 
-				fmt.Println("Namespace added: ", ns.Name)
```
Comment (gustavosbarreto) [inline cli/main.go:65]: why was this removed?

**Review by gustavosbarreto — APPROVED** [review]
